schemas/iso19139/src/main/plugin/iso19139/layout/tpl-rdf.xsl

```diff
@@ -187,6 +190,14 @@
             <xsl:value-of select="(gmd:protocol/gmx:Anchor)[1]"/>
           </dcat:mediaType>
           </xsl:if>
+
+          <!-- SIB addon-->
+          <xsl:if test="(gmd:description/gco:CharacterString)[1]!=''">
+          <dct:description>
+            <xsl:value-of select="(gmd:description/gco:CharacterString)[1]"/>
```

Comment:
note for $self: this is to solve `Description des données non renseignée` on harvested data, which comes from the attached resource description